account_product_fiscal_classification/model/account_product_fiscal_classification.py

```diff
@@ -44,6 +41,47 @@ def _default_company_id(self):
 
     description = fields.Text()
 
+    active = fields.Boolean(
+        default=True,
+        help="If unchecked, it will allow you to hide the Fiscal"
+        " Classification without removing it.")
+
+
+class AccountProductFiscalClassificationTemplate(models.Model):
```

Comment:
Could you set a file per model ? 

- OCA Convention 
- Code review will be more easy to do : For the time being github diff display that AccountProductFiscalClassification disappeared, replaced by AccountProductFiscalClassificationModel.